runtime-modules/proposals/engine/src/types/mod.rs

```diff
@@ -419,13 +379,64 @@ pub struct ProposalCreationParameters<BlockNumber, Balance, MemberId, AccountId>
     pub exact_execution_block: Option<BlockNumber>,
 }
 
+/// Contains staking balance and account id.
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Encode, Decode, Default, Clone, Copy, PartialEq, Eq, Debug)]
+pub struct Stake<AccountId, Balance> {
+    /// Staking account id.
+    pub account_id: AccountId,
+
+    /// Staking balance.
+    pub balance: Balance,
```

Comment:
If staking for proposals is exact, then why do we need this?

Reply:
It is an artifact. We could keep it as a possible agility feature in order to make a stake 'equal or greater than' preset someday or remove it for all proposals.

Reply:
No I don't think that is advisable, going to have to insist here.

Reply:
Sure.